mlxlink/modules/mlxlink_utils.cpp

```diff
@@ -93,7 +93,7 @@ string status2Color(u_int32_t status)
     return (status == 0 || status == 1023) ? ANSI_COLOR_GREEN : ANSI_COLOR_RED;
 }
 
-string getLoopbackColor(u_int32_t loopbackMode)
+string getLoopbackColor(int loopbackMode)
```

Comment:
What is the meaning of negative loopbackMode?
I took a look on the locations that it is used and seems that it should be unsigned.
For example:
setPrintVal(_operatingInfoCmd,PDDR_LOOPBACK_MODE , "Loopback Mode",
            _mlxlinkMaps->_loopbackModeList[loopbackMode],
            getLoopbackColor(loopbackMode), true, print);

Seems like you have a bug here if it's negative (-1) after it's set here:
int loopbackMode = (phyMngrFsmState != PHY_MNGR_DISABLED) ? getFieldValue("loopback_mode", _buffer) : -1;